### PR TITLE
fix: (Platform) Thumbnail accessibility bug fixes.

### PR DESCRIPTION
**Pull request by manu-kr**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/3825
https://github.com/SAP/fundamental-ngx/issues/3826
https://github.com/SAP/fundamental-ngx/issues/3828

#### Please provide a brief summary of this pull request.
This PR contains fixes for the accessibility bugs for thumbnail component. 
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.html [hunk 1]
```diff
@@ -6,9 +6,8 @@
         (click)="thumbnailClick(image)"
         (keydown.enter)="thumbnailClick(image)"
         tabindex="0"
-        role="button"
-        aria-pressed="false"
-        [title]="image.label"
+        [attr.title]="image.label"
+        [attr.ariaLabel]="image.label"
```
Comment (kavya-b) [inline libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.html:10]: Isn't it `attr.aria-label` ? Does this get read out by screen reader?

**Comment by kavya-b** [comment]
also, change PR title to correct format. `fix: (Platform) ...`

File: libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.html (hunk 1 above)
Comment (manu-kr) [reply libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.html:10]: yes its fixed now.

**Comment by manu-kr** [comment]
> also, change PR title to correct format. `fix: (Platform) ...`

modified.

**Review by mikerodonnell89 — APPROVED** [review]

**Review by kavya-b — APPROVED** [review]
Looks good. Just check or mark as `n/a` to the Documentation Checklist in PR description.

**Review by KevinOkamoto — APPROVED** [review]
Looks good.

**Pull request merged**